### PR TITLE
Suggest users to add GitHub labels to PRs

### PR DESCRIPTION
**Pull request by SeanNaren**
# What does this PR do ?

Suggests users to add labels to their PR. This helps classify for maintainers.

# Before your PR is "Ready for review"
**Pre checks**:
- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [ ] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -20,12 +18,9 @@ Add a one line overview of what this PR aims to accomplish.
...
 - [ ] Did you add or update any necessary documentation?
 - [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
-  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
-  
-**PR Type**:
-- [ ] New Feature
-- [ ] Bugfix
-- [ ] Documentation
+- [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
+- [ ] Did you add GitHub tags on the right? e.g Add `ASR`, `bug` for an ASR bugfix
```
Comment (SeanNaren) [inline .github/PULL_REQUEST_TEMPLATE.md:22]:
```suggestion
- [ ] Did you add GitHub labels on the right? e.g Add `ASR`, `bug` for an ASR bugfix
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -2,8 +2,6 @@
 
 Add a one line overview of what this PR aims to accomplish.
 
-**Collection**: [Note which collection this PR will affect]
```
Comment (titu1994) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: Let's keep collections please. A one line block determining collections is useful for the regex parser in the changelog builder

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -20,12 +18,9 @@ Add a one line overview of what this PR aims to accomplish.
 - [ ] Did you write any new necessary tests?
 - [ ] Did you add or update any necessary documentation?
 - [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
-  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
-  
-**PR Type**:
```
Comment (titu1994) [inline .github/PULL_REQUEST_TEMPLATE.md:25]: Please revert pr type changes. They are needed by the changelog builder

**Review by titu1994 — CHANGES_REQUESTED** [review]

**Comment by titu1994** [comment]
In general I think we need some way to auto label rather than ask users to label each pr (they can ofc do that if they want, but an automated solution is better when people don't want to do it)

**Comment by SeanNaren** [comment]
> In general I think we need some way to auto label rather than ask users to label each pr (they can ofc do that if they want, but an automated solution is better when people don't want to do it)

I think we should go straight to that if that's the case, if we have to keep "Collection" then we should auto-label :)

Will close this PR and investigate an auto-labeller!

**Pull request closed without merging**